### DISCUSSION TITLE
What's the license of Mortimer's Chip-disko?

### DISCUSSION THREAD

**Discussion by ghost** [SuperTux/supertux#2825] [General]
I am creating album with various (not mine) house songs that are free (libre), then, I played SuperTux and Mortimer's Chip-disko sounds like house, I'd like to add it to my album, but I don't know what license it uses. (I know how to create house, but look at ZaurOrg, it has record label)

**Comment by ghost** [comment]
Oh, I will turn 20 in 2 days(0.0.6 was released on March 15 2004)

**Reply by ghost** [reply]
I'm not 20, Mortimer's Chip-disko is 20

**Reply by jamescdericco** [reply]
Oh, thanks for the clarification! That's an interesting fact.

**Comment by jamescdericco** [comment]
Hello @GNUAn, Mortimer's Chip Disko (chipdisko.ogg) is licensed under the CC-BY-SA and GPL by Lukas Nystrand (as Mortimer Twang). I don't know which specific license versions are used. There is licensing info for the music in the [data/AUTHORS file](https://github.com/SuperTux/supertux/blob/master/data/AUTHORS).

**Reply by ghost** [reply]
Oops, i wanted to say 'It', not 'I'

**Reply by ghost** [reply]
But thanks, now i can add it to my album
